app/furniture/marketplace/carts/_cart.html.erb

```diff
@@ -4,6 +4,7 @@
     <%= form_with model: cart.location do |cart_form| %>
       <%= render "text_field", attribute: :delivery_address, form: cart_form %>
       <%= render "datetime_field", attribute: :delivery_window, form: cart_form, disabled?: cart.marketplace.delivery_window.present? %>
+      <%= render "email_field", attribute: :contact_email, form: cart_form %>
```

Comment:
Happy to split this and the model changes into their own PR where we start collecting the `Order#contact_email` independent of sending the `Order::PlacedMailer`